application/backend/src/main/resources/apps/typerefinery/web_root/main.css

```diff
@@ -9,6 +9,15 @@ body {
   width: 100%;
   margin-bottom: 1%;
 }
+
+.lineChart,
+.barCanvas,
+.pieCanvas,
+.xAxisCanvas {
```

Comment:
you can just use one single class

"chartCanvas"

Reply:
ok